microsoft-edge/devtools-guide-chromium/console/console-dom-interaction.md

```diff
@@ -5,290 +5,345 @@ author: MSEdgeTeam
+   ![The Elements tool in DevTools, with the link element selected](./console-dom-interaction-images/link-in-elements-tool.png)
 
-1. Right-click the element in the DOM tree and then select **Copy** > **Copy JS Path**.
+1. Click the `...` button next to the element in the DOM tree and then select **Copy** > **Copy JS Path**.
 
-   ![Copy the JavaScript path from an element in the DOM tree of the Elements tool](./console-dom-interaction-images/console-dom-copy-JS-path.png)
+   ![The Copy JS Path menu item in the Elements tool](./console-dom-interaction-images/copy-js-path-menu.png)
 
-   <!-- could be useful to have code listings.  test this:
    Here's the JS path you copied:
 
    ```javascript
-   document.querySelector("#headerAreaHolder > header > div:nth-child(1) > div.nav-bar-item.is-hidden-tablet > div > button > span")
+   document.querySelector("body > main > section:nth-child(1) > ul > li:nth-child(1) > a")
    ```
 
-   Here's the result after you add text content:
-
-   ```javascript
-   document.querySelector("#headerAreaHolder > header > div:nth-child(1) > div.nav-bar-item.is-hidden-tablet > div > button > span").textContent = "My Playground"
-   ``` -->
+   The above path is a CSS selector that points to the link element you selected by using the **Inspect** tool.
 
 1. In the **Console**, paste the JavaScript path that you copied, but don't press **Enter** yet.
 
-1. Change the text of the link to `My Playground`.  To do that, add `.textContent = "My Playground"` to the JavaScript path that you previously pasted:
+1. Change the text of the link to `Updated link`.  To do that, add `.textContent = "Updated link"` to the JavaScript path that you previously pasted:
 
-   ![Using the Console to change the content of an element](./console-dom-interaction-images/console-dom-change-content.png)
+   ![The Console tool, with the JS path and textContent expression, and the webpage, showing the updated link text](./console-dom-interaction-images/updated-link-text.png)
 
-Use any JavaScript DOM manipulations you want in the **Console**.  To make it more convenient, the **Console** comes with a few helper utility methods.
+Use any DOM APIs you want to change the DOM tree in the **Console**.  To make it more convenient, the **Console** comes with a few helper utility methods.
 
 
 <!-- ====================================================================== -->
-## Helpful Console utility methods
+## Helpful console utility functions
 
-Many convenience methods and shortcuts are available to you as [Console Utilities](utilities.md).  Some of the methods are incredibly powerful and are much more effective than using `console.log()` statements.
+Many convenience methods and shortcuts are available to you as utility functions in the **Console** tool. Some of these utility functions are very useful for interacting with the DOM tree of a webpage and are described below. To learn more, see [Console tool utility functions and selectors](./utilities.md).
```

Comment:
```suggestion
Many convenience methods and shortcuts are available to you as utility functions in the **Console** tool. Some of these utility functions are very useful for interacting with the DOM tree of a webpage, and are described below. To learn more, see [Console tool utility functions and selectors](./utilities.md).
```
* comma separating two distinct points, to dole out 1 at a time for easy parsing